src/ssh-util/src/tunnel.rs

```diff
@@ -68,6 +70,14 @@ impl fmt::Debug for SshTunnelConfig {
     }
 }
 
+#[derive(Clone)]
+pub enum SshTunnelStatus {
+    Running,
+    Errored(String),
+}
```

Comment:
If you're adding new public API to a crate, it needs to come with docstrings!

Reply:
Looking at this more though, I'm not sold on this needing to be a dedicated type! Seems like `Result<(), String>` would do the trick here?

Reply:
On third thought, I think the explicitness of this type is nice. But please add docstrings!

Reply:
yeah, the explicitness is really nice